### ISSUE TITLE
Create a responsive Home Page.

### ISSUE THREAD

**Issue by aakankshabhende** [aakankshabhende/The-Cafeteria#1]

**Comment by everly-gif** [comment]
Hi ! @aakankshabhende I can take this up ,  what exactly are you planning for the homepage to look like , any design in your mind?

**Comment by aakankshabhende** [comment]
Yes @everly-gif you can contribute to this repository if you are a participant of DWoC'21. I don't have any design in mind, you are free to make one your way :)

**Comment by everly-gif** [comment]
yes @aakankshabhende I'm a participant of DWoC'21 and I will get going with the homepage. Btw what are the technologies I will get to work on if I submit a proposal and get selected ? It will be really great to learn react and node from scratch. Is that what you plan to do?

**Comment by aakankshabhende** [comment]
I'm planning to keep it simple and not go into the MERN stack so even the newbies can help in contributing this project and start their open source journey @everly-gif

**Comment by harshit995** [comment]
pls assign it to me as well

**Comment by aniket7515** [comment]
I am also interested in contributing to the project
I am a DWoC participant

**Comment by aakankshabhende** [comment]
You can start working on it and the best homepage will be selected. @aniket7515 @harshit995 .I'll assign this issue after coding period will officially begin.

**Comment by charu210703** [comment]
Can you tell me what are the features you want in this homepage?

**Comment by devesh2511** [comment]
I'm interested in working on the front end part of the homepage, please let me know how can I contribute?

**Comment by aakankshabhende** [comment]
> Can you tell me what are the features you want in this homepage?

We will start from simple like FAQ page, cart, payment page, about us, contact us, etc @charu210703

**Comment by aakankshabhende** [comment]
> I'm interested in working on the front end part of the homepage, please let me know how can I contribute?

Are you DWOC participant? @devesh2511

**Comment by devesh2511** [comment]
> > I'm interested in working on the front end part of the homepage, please let me know how can I contribute?
> 
> Are you DWOC participant? @devesh2511

Yes

**Comment by charu210703** [comment]
I am DWoC participant. I have submitted the proposal with some additional features too. Can you look into it?
